dtfs-central-api/src/v1/controllers/utilisation-report-service/fee-record-correction/get-fee-record-correction-request-review.controller/index.ts

```diff
@@ -17,17 +17,19 @@ export type GetFeeRecordCorrectionRequestReviewRequest = CustomExpressRequest<{
 /**
  * Response body type for the GET fee record correction request review route.
  */
-export type GetFeeRecordCorrectionRequestReviewResponseBody = {
-  bank: SessionBank;
-  reportPeriod: ReportPeriod;
-  correctionRequestDetails: {
-    facilityId: string;
-    exporter: string;
-    reasons: RecordCorrectionReason[];
-    additionalInfo: string;
-    contactEmailAddresses: string[];
-  };
-};
+export type GetFeeRecordCorrectionRequestReviewResponseBody =
+  | {
+      bank: SessionBank;
+      reportPeriod: ReportPeriod;
+      correctionRequestDetails: {
+        facilityId: string;
+        exporter: string;
+        reasons: RecordCorrectionReason[];
+        additionalInfo: string;
+        contactEmailAddresses: string[];
+      };
+    }
+  | { errorKey: string };
```

Comment:
Should we introduce a type that extracts 'INVALID_STATUS' from the const and then use that here so only that specific string can be set?